schema/deploy/shipping/views.sql

```diff
@@ -131,6 +131,8 @@ grant select
     on shipping.presence_absence_result_v1
     to "incidence-modeler";
 
+
+drop view shipping.incidence_model_observation_v2;
 create view shipping.incidence_model_observation_v2 as
```

Comment:
I prefer to use `create or replace view` statements when we can get away with them (when we're not re-ordering or re-typing columns), as they avoid some problems with inter-dependent views.